explorer/lib/explorer_web/components/nav.ex

```diff
@@ -109,4 +122,16 @@ defmodule NavComponent do
     |> JS.toggle(to: ".toggle-open")
     |> JS.toggle(to: ".toggle-close")
   end
+
+  defp active_view_class(ExplorerWeb.Home.Index, ExplorerWeb.Home.Index), do: "text-4xl"
+  defp active_view_class(_other, ExplorerWeb.Home.Index), do: "text-3xl"
+
+  defp active_view_class(ExplorerWeb.Batches.Index, ExplorerWeb.Batches.Index ), do: "text-green-500 font-bold"
+  defp active_view_class(ExplorerWeb.Batch.Index, ExplorerWeb.Batches.Index ), do: "text-green-500 font-bold"
+  defp active_view_class(_other, ExplorerWeb.Batches.Index ), do: "text-foreground/80 hover:text-foreground font-semibold"
+
+  defp active_view_class(ExplorerWeb.Operators.Index, ExplorerWeb.Operators.Index ), do: "text-green-500 font-bold"
+  defp active_view_class(ExplorerWeb.Operator.Index, ExplorerWeb.Operators.Index ), do: "text-green-500 font-bold"
+  defp active_view_class(_other, ExplorerWeb.Operators.Index ), do: "text-foreground/80 hover:text-foreground font-semibold"
```

Comment:
I would avoid pattern matching here. You end up repeating the styles all over and it make testing and updating each case very difficult.

Reply:
What alternative do you suggest?
I think the only problem with the current solution is that uncatched cases are not identifiable (i.e. the restakings page)

Reply:
For reducing styling duplicates, maybe just use a separate CSS class and return only that class might work, right? I mean, with the exception of the logo everything follows the pattern "if active make green and bold, otherwise default text-foreground and semibold".

Then, we can trim the cases a bit by considering that the only same-class pair that does not follow this pattern is the logo, so everything else could be covered by a single pattern:
```elixir
defp active_view_class(class_x, class_x), do: "active-view"
```
Similarly, our fallthrough to non-highlighter would be:
```elixir
defp active_view_class(class_x, class_y), do: "active-view"
```

Reply:
At the end I think we would have something like this, if applying this idea:
```elixir
defp active_view_class(ExplorerWeb.Home.Index, ExplorerWeb.Home.Index), do: "active-view aligned-icon"
defp active_view_class(_other, ExplorerWeb.Home.Index), do: "inactive-view aligned-icon"
defp active_view_class(class_x, class_x), do: "active-view"
defp active_view_class(ExplorerWeb.Operator.Index, ExplorerWeb.Operators.Index), do: "active-view"
defp active_view_class(ExplorerWeb.Batch.Index, ExplorerWeb.Batches.Index), do: "active-view"
defp active_view_class(class_x, class_y), do: "inactive-view"
```

Reply:
How much do we actually gain with this? I don't find the last solution significantly clearer than the current one. Also, we are only reducing two lines, but now we need to add four new class definitions. That said, I'm open to changing it if you think it's worthwhile.